apps/docs/src/app/core/component-docs/moment-datetime-adapter/examples/moment-adaptor-options-example.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+import { DatetimeAdapter, MOMENT_DATE_TIME_ADAPTER_OPTIONS, MomentDatetimeAdapter } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fd-moment-adaptor-options-example',
+    templateUrl: './moment-adaptor-options-example.component.html',
```

Comment:
The template isn't set inline and it's not shown on the separate tab in examples, when opening StackBlitz template file (`.html`) isn't loaded also.